prisma/migrations/20230912072503_add_associated_designs_table/migration.sql

```diff
@@ -0,0 +1,16 @@
+-- CreateTable
+CREATE TABLE "AssociatedDesign" (
```

Comment:
/nit probs too late to change this for existing tables already created (or is it?) but postgres table naming convention is snake case not camel case (reason being that its actually case insensitive so a lot of editors default lower case everything making it hard to read without snake case)